soc/nordic/nrf54h/Kconfig

```diff
@@ -63,6 +63,13 @@ config SOC_NRF54H20_CPURAD_COMMON
 	select HAS_PM
 	select HAS_POWEROFF
 
+config SOC_NRF54H20_CPURAD_ENABLE
```

Comment:
Is it idiomatic to have the `_ENABLE` suffix here? Having `_ENABLE` suffix in a kconfig option is redundant IMO as you are "enabling" an option in the first place.

Reply:
It is bad practice. But I want to be consistent with SOC_NRF53_CPUNET_ENABLE.

Reply:
I suppose omitting _ENABLE might make it sound like the option indicates whether this is a NET CPU.